src/app/lib/config/services/afaanoromoo/translations.ts

```diff
@@ -0,0 +1,215 @@
+export default {
+  pagination: {
+    previousPage: 'Kan duraa',
+    nextPage: 'Itti aanee',
+    pageXOfY: 'Page {x} Keessaa {y}',
+  },
+  ads: {
+    advertisementLabel: 'Beeksiisa',
+  },
+  seeAll: 'Hunda ilaali',
+  home: 'Oduu',
+  currentPage: 'Fuula kan ammaa',
+  skipLinkText: 'Qabiyyeetti darbi',
+  relatedContent: 'Odeessa kana irratti dabalata',
+  relatedTopics: 'Mata dureewwan walitti dhiyaatan',
+  navMenuText: 'Kutaawwan',
+  mediaAssetPage: {
+    mediaPlayer: 'Media player',
+    audioPlayer: 'Audio player',
+    videoPlayer: 'Video player',
+  },
+  liveExperiencePage: {
```

Comment:
Where do these translations come from - are they somehow autogenerated / pulled in from iSite, or am I getting confused?